secrets/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "doc.go",
+        "errors.go",
+        "secrets.go",
+        "store.go",
+    ],
+    importpath = "github.com/reddit/baseplate.go/secrets",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//batcherror:go_default_library",
+        "//filewatcher:go_default_library",
+        "//log:go_default_library",
+    ],
+)
+
+go_test(
+    name = "go_default_test",
+    srcs = [
+        "secrets_test.go",
```

Comment:
How does this work, I assume the list in here gets outgenerated?

Reply:
yes, that's auto generated by gazelle, by running:

```
bazel run //:gazelle
```